src/app/pages/profile/transaction-information/transaction-information.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { TransactionInformationService } from '../../../services/profile/transaction-information/transaction-information.service';
+import { APPCONFIG } from 'src/app/config';
+
+@Component({
+  selector: 'app-transaction-information',
+  templateUrl: './transaction-information.component.html',
+  styleUrls: ['./sass/transaction-information.component.scss']
+})
+export class TransactionInformationComponent implements OnInit {
+  base_url = APPCONFIG.base_url;
```

Comment:
Kindly take advantage of the environment files that come with Angular to import the base url

Reply:
Thanks @JamesMudidi  for your feedback.... I'm of course using the environment file. Kindly check it out